src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentTypeBuilder.cs

```diff
@@ -23,11 +23,10 @@ public void Build(FieldType contentQuery, ContentTypeDefinition contentTypeDefin
 
             foreach (var part in contentTypeDefinition.Parts)
             {
-                // Check if another builder has already added a field for this part.
-                if (contentItemType.HasField(part.Name)) continue;
+                var partName = part.Name;
 
-                // This builder only handles parts with fields.
-                if (!part.PartDefinition.Fields.Any()) continue;
```

Comment:
I don't yet understand why this check needs to be removed. The code in this class only operates on fields, so if the part has no fields we probably shouldn't create an empty field?

Reply:
Yes, it still works if i put it back, maybe i needed it at some point. i will revert this change.